src/fontstash.h

```diff
@@ -1007,20 +1079,34 @@ static void fons__blur(FONScontext* stash, unsigned char* dst, int w, int h, int
 }
 
 static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned int codepoint,
-								 short isize, short iblur)
+								 short isize, short iblur, int oversampleH, int oversampleV)
 {
 	int i, g, advance, lsb, x0, y0, x1, y1, gw, gh, gx, gy, x, y;
 	float scale;
 	FONSglyph* glyph = NULL;
 	unsigned int h;
 	float size = isize/10.0f;
+	float subX, subY;
+	float recipH, recipV;
 	int pad, added;
 	unsigned char* bdst;
 	unsigned char* dst;
 
 	if (isize < 2) return NULL;
 	if (iblur > 20) iblur = 20;
-	pad = iblur+2;
+	if (oversampleH < 1) oversampleH = 1;
+	if (oversampleV < 1) oversampleV = 1;
+
+	// Only use extra 1px "leak" padding if blurring (otherwise it breaks oversampling)
```

Comment:
I remember adding that second padding pixel because rotated text would not work otherwise. I recommend to change the padding to 1 and try rotated text. Test snapped, oversampled, and blurred. If it works, then the problem was somewhere else and we can keep just the 1px padding. If it does not, then we'll need more padding (yeah, sucks).